Linux/install-vrcx.sh

```diff
@@ -78,6 +78,15 @@ export WINEPREFIX=$WINEPREFIX
 wine $WINEPREFIX/drive_c/vrcx/VRCX.exe" > $WINEPREFIX/drive_c/vrcx/vrcx
 chmod +x $WINEPREFIX/drive_c/vrcx/vrcx
 
+rel_path=$(dirname $(realpath $0))
+
+# Install twemoji fonts as Segoe UI is proprietary and not included in wine
+if [[ -d "$rel_path/fonts" ]]; then
+    echo "Installing twemoji fonts."
+    cp -r "$rel_path/fonts/"* "$WINEPREFIX/drive_c/windows/Fonts"
+	WINEPREFIX=$WINEPREFIX wine reg add 'HKLM\Software\Microsoft\Windows NT\CurrentVersion\Fonts' /v 'seguiemj' /t REG_SZ /d 'seguiemj.ttf' /f
+fi
```

Comment:
If I understand this correctly, doesn't this depend on a user pulling the VRCX repo? I think most users only execute the script directly as per curl, think that'd never pull the fonts folder.

Reply:
Good catch, I will update the script to curl the font file from the repo.